azkaban-web-server/src/test/java/azkaban/flowtrigger/TriggerInstanceProcessorTest.java

```diff
@@ -86,7 +91,7 @@ public void testProcessTermination() throws ExecutorManagerException, Interrupte
     final TriggerInstance triggerInstance = createTriggerInstance();
     processor.processTermination(triggerInstance);
     sleep(1000);
-    verify(emailer).sendEmail(any(), any(), any());
+    verify(emailer).sendEmail(any(), anyBoolean(), anyString());
```

Comment:
How about using this strategy to check the output:
https://github.com/azkaban/azkaban/blob/master/azkaban-common/src/test/java/azkaban/utils/EmailerTest.java#L97-L98
?

When you have the expected output saved as a .html file, you can easily render it in browser to check that it looks as expected, like this one:
https://github.com/azkaban/azkaban/blob/master/azkaban-common/src/test/resources/azkaban/executor/mail/errorEmail2.html